samples/snippets/visualbasic/VS_Snippets_VBCSharp/VbSimpleQuerySamples/VB/Data/Customers.xml

```diff
@@ -45,8 +45,8 @@
   <customer>
     <id>ANATR</id>
     <name>Ana Trujillo Emparedados y helados</name>
-    <address>Avda. de la Constitución 2222</address>
-    <city>México D.F.</city>
+    <address>Avda. de la Constituci?n 2222</address>
+    <city>M?xico D.F.</city>
```

Comment:
It looks like something messed up the characters encoded here. Guessing it wasn't intended?

Reply:
This was done by mistake, and since removed from this PR.